frontend/src/exams/exams.vue

```diff
@@ -8,15 +8,16 @@
 </template>
 
 <script>
+  import { mapActions } from 'vuex'
   import ExamInventoryTable from './exam-inventory-table'
 
   export default {
     name: "Exams",
-    components: { ExamInventoryTable }
+    components: { ExamInventoryTable },
```

Comment:
Why the comma?

Reply:
IDE probably added that, or I may have.  I prefer trailing commas and generally always use them on anything comma separated listed things i write as when I don't do this and find the need to add another item to the list I almost alway forget to add the coma to the previous item so it saves me time.